docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -68,8 +68,8 @@ public static void Example()
                     batchSize: 10,
                     learningRate: 0.01f,
                     metricsCallback: (metrics) => Console.WriteLine(metrics),
-                    validationSet: testDataset);
-
+                    validationSet: testDataset)
+                    .Append(mlContext.Transforms.Conversion.MapKeyToValue(outputColumnName: "PredictedLabel", inputColumnName: "Label"));
```

Comment:
>outputColumnName: "PredictedLabel", inputColumnName: "Label") [](start = 74, length = 61)

This is incorrect. What it will do is create a new column called `PredictedLabel` (overwriting the one created by the `ImageClassificationTransformer`), that is the key to value transformer applied to the `Label` column.

Reply:
It is enough to just pass one string: "PredictedLabel" (when `inputColumnName` is not passed, `outputColumnName` is used for both output and input).

---
In reply to: [327501425](https://github.com/dotnet/machinelearning/pull/4228#discussion_r327501425) [](ancestors = 327501425)